hack/resolve-etcd-version-from-etcd-druid.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+#
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
Quite old script ;-) ...

Reply:
I'm getting tired of this. Should we just remove these year numbers from all files? Some files have them, others don't, and we don't have a CI check to ensure they are set correctly (i.e., if a human misses it in the PR review, it's "messed up"). I don't see a big value since nobody really cares about in which year a particular file was newly created. Thoughts?

Reply:
My understanding from the last discussion with @RaphaelVogel was that the year numbers are not important and can be removed.